.github/workflows/sink.yml

```
@@ -13,22 +13,20 @@ on:
     - main
     tags: [ '*.*.*' ]
     paths:
-    - 'hack/e2e/sink/**'
-    - '.github/workflows/sink.yml'
-    paths-ignore:
-      - 'docs/**'
-      - '**.md'
-      - 'sec-scanners-config.yaml'
+      - 'hack/e2e/sink/**'
+      - '.github/workflows/sink.yml'
+      - '!docs/**'
+      - '!**.md'
+      - '!sec-scanners-config.yaml'
   pull_request:
     branches:
     - main
     paths:
-    - 'hack/e2e/sink/**'
-    - '.github/workflows/sink.yml'
-    paths-ignore:
-      - 'docs/**'
-      - '**.md'
-      - 'sec-scanners-config.yaml'
+      - 'hack/e2e/sink/**'
+      - '.github/workflows/sink.yml'
+      - '!docs/**'
+      - '!**.md'
+      - '!sec-scanners-config.yaml'
```

I think for this test it is sufficient to only have path define and to not define anything that should not be scanned.
```suggestion
      - 'hack/e2e/sink/**'
      - '.github/workflows/sink.yml'
```